addons/udes_security/controllers/main.py

```diff
@@ -77,7 +77,10 @@ def _get_menu_info_from_action_id(self, action_id):
             try:  # If you middle click a menu, action_id is a string..
                 action_id = int(action_id)
             except ValueError:
-                raise ValidationError("Invalid action ID specified.")
+                try:  # JS `do_action("action_extid")` results in the parameter coming in as a string here
```

Comment:
It's probably not an issue but wouldn't it be better to have a separate if statement and try and use the string and except separately?

Reply:
we only want to run the second check if it fails the int pass, because it's more performant. The int could be wrapped in a string like "11" so can't do `type()` or `isinstance()` in an if check, instead we just have to let it fail and only if it fails conversion run this second pass